wandb/integration/openai/__init__.py

```diff
@@ -1,3 +1,17 @@
-__all__ = ("autolog",)
+__all__ = ("autolog", "WandbLogger")
 
-from .openai import autolog
+try:
+    import openai
+except Exception:
+    print(
+        "Error: `openai` not installed >> This integration requires openai!  To fix, please `pip install openai`"
+    )
```

Comment:
you could use our built-in `get_module` logic see in wandb/util.py` it will do all of this logic for you. 
Only thing to note is that it is lazy loader by default, something to keep in mind!